README.md

```diff
@@ -63,7 +61,7 @@ We recommend connecting the **Slack** integration.
 
 3. **Enjoy not having to manually clean up pull request branches when PRs are closed!**
 
-    Note: the label is automatically added to and removed from the repository depending. 
+**Note:** this skill automatically adds the labels needed for auto-deletion. 
```

Comment:
The Note in the auto-merge skill README didn’t get bolded. I’d like to keep this consistent across all READMEs. Your call of you want it in bold or not.

Reply:
Okay. I'll undo the bold here for consistency. At some point, I think we'll want to have a markdown style that renders in these pages as more of a tip/callout, but will wait until we design that.